satpy/readers/ahi_hsd.py

```diff
@@ -583,6 +612,15 @@ def convert_to_radiance(self, data):
         else:
             gain = self._header["block5"]["gain_count2rad_conversion"][0]
             offset = self._header["block5"]["offset_count2rad_conversion"][0]
+        # If using custom calibration from GSICS, apply it here
+        if self.calib_mode == "CUSTOM":
+            if bname in self.custom_calib:
+                g2 = self.custom_calib[bname][0]
+                o2 = self.custom_calib[bname][1]
+                data = (data - o2) / g2
```

Comment:
I'm not sure I understand what is happening here, it seems like the calibration is reversed? Why not just replace gain and offset and let the current code to the rest?

Reply:
See my earlier comment to you and @sfinkens. These calibrate the radiances, not the digital numbers. In other words, I'm applying a second, additional, calibration on top of the original slope + offset calibration.

Reply:
Yes, sorry I saw that after the fact. Expressing them in terms of gain and offset would help though creating that new function I'm talking about, right?